.github/workflows/ci.yml

```diff
@@ -690,7 +690,7 @@ jobs:
   tests-cross-language:
     name: Cross-Language Tests
     needs: [ java-app-encryption, csharp-app-encryption, go-app-encryption, java-server, go-server ]
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Could we use the `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT` to fix this and keep `ubuntu-latest` for consistency?

Reply:
We could... maybe. Taking another look, I'm thinking all we need is to change the `dotnet-version` to `6.0.x` in the `Set up C#` step. I'll try that (and revert back to ubuntu-latest).